README.md

```diff
@@ -62,12 +65,22 @@ A simplified example DAG demonstrates how to use the [Airflow Operator](/example
 from airflow_provider_hex.operators.hex import HexRunProjectOperator
 
 PROJ_ID = 'abcdef-ghijkl-mnopq'
+notifications: list[NotificationDetails] = [
+    {
+        "type": "SUCCESS",
+        "includeSuccessScreenshot": True,
+        "slackChannelIds": ["HEX666SQG"],
+        "userIds": [],
+        "groupIds": [],
+    }
+]
 ...
 sync_run = HexRunProjectOperator(
     task_id="run",
     hex_conn_id="hex_default",
     project_id=PROJ_ID,
     dag=dag,
-    input_parameters={'myParam': 42}
+    input_parameters={'myParam': 42},
```

Comment:
```suggestion
```

do we want to remove the input param from the readme here, given invalid params cause the run to return an errored response status code? that might be confusing for someone starting out.